src/Controls/ColumnChooser.xaml.cs

```diff
@@ -831,4 +834,117 @@ public bool Filter(object data, string searchString)
             return data is DimensionColumnData;
         }
     }
+
+    public class Win32Helper_dav
```

Comment:
please add this into a gernal helper.cs / tools.cs or Scaling.cs,... and don't mix it

Reply:
this service comes from the  q2g-helper-sense-nuget.
to fix that ugly workaround.
* Can I move the class to leonardo-wpf as q2g-helper-sense-nuget references on it already?